docker/entrypoint.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+if [ ! -f /db/db.sqlite3 ] ; then
+  touch /db/db.sqlite3
+fi
+ln -s /db/db.sqlite3 /dakara-server/dakara_server/db.sqlite3
+
+if [ "$1" == "install" ]; then
+  /dakara-server/dakara_server/manage.py migrate
+  /dakara-server/dakara_server/manage.py createsuperuser
```

Comment:
You need to create the player.

Reply:
I honestly never noticed there was a command for this (I have always created the player from the web interface...)